Assets/MixedRealityToolkit/BuildAndDeploy/Editor/BuildDeployPreferences.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using MixedRealityToolkit.Build.WindowsDevicePortal.DataStructures;
```

Comment:
Does us moving to requiring .net 4.6 allow us to use the existing Windows Device Portal Wrapper instead of having to clone the functionality?

https://github.com/Microsoft/windowsdeviceportalwrapper

Reply:
Might not be worth it, seems that project is full of problems (https://github.com/Microsoft/WindowsDevicePortalWrapper/issues?utf8=✓&q=is%3Aissue+is%3Aopen+hololens)

Reply:
I thought about using it, but didn't want to put another external dependency on this repository.

Reply:
I'd rather see a Unity specific solution to the Windows Device Portal.

Reply:
@davidkline-ms These issues date almost 2 years back, with no resolution, and afflict fundamentals such as deployment and app launching. That's a giant red flag, in my opinion. Just my observation, you guys are the experts and know better.